README.org

```diff
@@ -1,45 +1,49 @@
-#+TITLE: Opencl-mode
-#+AUTHOR: BAH SALMANE
-#+EMAIL: salmane.bah@u-bordeaux.fr
-#+DATE: 25-06-2014
+#+TITLE: OpenCL Major Mode
+#+AUTHOR: Gustaf Waldemarson
+#+EMAIL: gustaf.waldemarson ~at~ gmail.com
```

Comment:
And email addresses

Reply:
Hello!

Nice thing that you caught all the author changes, I had forgotten that I made those a while back when I was thinking I might have to take over ownership of the repo, which is clearly not necessary!

All things you have noted above have been fixed, however the: `(define-key opencl-mode-map (kbd "C-c ! d") 'opencl-lookup)` is now unnecessary, as it is added directly to the map when it is being defined on lines 713-717.